src/main/java/com/frederikam/fred/moe/SpringController.java

```diff
@@ -79,6 +79,7 @@ public void get(HttpServletRequest request, HttpServletResponse response) throws
                 TikaInputStream.get(f.toPath()), metadata);
 
         response.setContentType(mediaType.toString());
+        response.setContentLengthLong(f.length());
```

Comment:
This is a Servlet 3.1 method, but the embedded tomcat supports that.
Just wanted you to know @Frederikam.